### ISSUE TITLE
Localize Wikipedia links

### ISSUE THREAD

**Issue by AutomCoding** [dessalines/lemmy#243]
There are two links to Wikipedia in this string.

https://github.com/dessalines/lemmy/blob/406bb14a4858fe322a77e7e588e6ad8833d6968d/ui/src/translations/en.ts#L137

There is an article about the Fediverse in many of the languages Lemmy supports, so that link could in most cases be translated to point to the language version of the translation.

The first link should probably point to the article on [social network aggregation](https://en.wikipedia.org/wiki/Social_network_aggregation), instead of [link aggregation](https://en.wikipedia.org/wiki/Link_aggregation).

**Comment by dessalines** [comment]
Question: If you browser language is set to german, but you view a page on `en` wikipedia that also exists on `de`, does it offer you a redirect?

The other thing that makes me hestitant to do anything about this is that all the other links go to projects that unfortunately are in english, even the README here is only in english.

**Comment by AutomCoding** [comment]
No, you have to scoll down to click the language link in the sidebar (which is often hidden by a more-button on pages with more than 10 languages).

![wp-langs](https://user-images.githubusercontent.com/40275136/64030453-e87a7700-cb46-11e9-8420-cf198289daaf.png)

**Comment by dessalines** [comment]
Ah. Okay I've changed it to social network aggregation at least, but am not sure I wanna redirect the other links, since their all in english, here's the current line:

```html
<T i18nKey="landing_0">#<a href="https://en.wikipedia.org/wiki/Social_network_aggregation">#</a><a href="https://en.wikipedia.org/wiki/Fediverse">#</a><br></br><code>#</code><br></br><b>#</b><br></br><a href={repoUrl}>#</a><br></br><a href="https://www.rust-lang.org">#</a><a href="https://actix.rs/">#</a><a href="https://infernojs.org">#</a><a href="https://www.typescriptlang.org/">#</a>
```

If you can find a good way to do this let me know.

**Comment by AutomCoding** [comment]
The easiest way to do it is passing the whole URL to the translators as a string to be "translated". This is the approach taken by most other projects I have encountered, and what I would recommend.

An overengineered way is to query Wikidata (the database where interwiki data for the Wikimedia projects is stored) for the URLs, with fallback to English. I did this in PHP a few years ago for another project (which had hundreds of Wikipedia links). The obvious downside of this approach is that it will only work for Wikipedia links, but they will always be up to date with articles being renamed and created in new languages. For two URLs per language, I'd say it is not worth the implementation time.
